mip/develop/concept-delegation.md

```diff
@@ -123,25 +123,25 @@ var protectionHandler = engine.CreateProtectionHandlerForConsumption(consumption
 
 #### C++
 
-This sample demonstrates how to perform **delegated consumption**.
+This sample demonstrates how to perform **delegated publishing**.
```

Comment:
I think these are mixed up. The example headers were switched and said consumption for publishing and vice versa. This looks like you updated the sample code and then reversed the headers. We can either switch the headers only or leave the headers and take the sample code updates.